v3/newrelic/log_events.go

```diff
@@ -181,6 +181,14 @@ func (events *logEvents) CollectorJSON(agentRunID string) ([]byte, error) {
 			}
 		}
 	}
+	if events.config.customAttributes != nil {
+		for k, v := range events.config.customAttributes {
+			buf.WriteByte(',')
+			jsonx.AppendString(buf, "tags."+k)
```

Comment:
Is this what the spec for custom attributes is asking for? I think the `tags.` prefix was specifically for labels. custom log events would be something else, otherwise the feature would be identical to labels, wouldn't it?

Reply:
There is no mention of a prefix in the [spec](https://source.datanerd.us/agents/agent-specs/pull/709/files) to custom attributes.